multicast/__main__.py

```
@@ -831,10 +831,10 @@ def cli() -> int:
 	__EXIT_CODE: tuple = (1, exceptions.EXIT_CODES[1][1])
 	if (sys.argv is not None) and (len(sys.argv) > 1):
 		__EXIT_CODE = main(sys.argv[1:])
-	elif (sys.argv is not None):
+	elif (sys.argv is not None):  # pragma: no branch
 		__EXIT_CODE = main([__name__, "-h"])
 	return __EXIT_CODE[0]
 
 
 if __name__ in '__main__':
-	exit(cli())  # skipcq: PYL-R1722 - intentionally allow overwriteing exit for testing
+	exit(cli())  # skipcq: PYL-R1722 - intentionally allow overwriting exit for testing
```

Pylint (3.3.7)

[warning] Bad indentation. Found 1 spaces, expected 4

(W0311)